scripts/build_docker_image.sh

```diff
@@ -57,6 +57,10 @@ go get golang.org/x/lint/golint
 mv $GOPATH/bin/golint /usr/local/bin
 go get golang.org/x/tools/cmd/goyacc
 mv $GOPATH/bin/goyacc /usr/local/bin/
+go get golang.org/x/tools/cmd/cover
```

Comment:
Do we need to install this? I think `cover` is part of the gotools.

Reply:
This is following https://docs.coveralls.io/go, I think it's better to keep it here.